src/sidebar/components/excerpt.js

```diff
@@ -20,7 +20,8 @@ function InlineControls({ isCollapsed, setCollapsed, linkStyle = {} }) {
         <button
           className="excerpt__toggle-button"
           onClick={() => setCollapsed(!isCollapsed)}
-          aria-label="Toggle to show the full excerpt or first few lines only"
+          aria-expanded={!isCollapsed}
+          aria-label="Toggle to show the full excerpt"
           aria-pressed={(!isCollapsed).toString()}
```

Comment:
While we're touching this, I think can convert a few of these over. I think

`aria-pressed={!isCollapsed}`

https://github.com/preactjs/preact/pull/2347

Reply:
Feel free to do so! I really want to focus just on labels and titles in this PR.